packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -57,9 +58,14 @@ contract L1Block is ISemver, IGasToken {
     /// @notice The latest L1 blob base fee.
     uint256 public blobBaseFee;
 
-    /// @custom:semver 1.4.1-beta.1
+    /// @notice Storage slot that the isDeposit is stored at.
+    ///         This is a custom slot that is not part of the standard storage layout.
+    /// keccak256(abi.encode(uint256(keccak256("l1Block.identifier.isDeposit")) - 1)) & ~bytes32(uint256(0xff))
+    uint256 internal constant IS_DEPOSIT_SLOT = 0x921bd3a089295c6e5540e8fba8195448d253efd6f2e3e495b499b627dc36a300;
```

Comment:
ah, is there a way to do this without hard-coding the hash? (all good if not), but avoids having to double check the calculations.

Reply:
also, this should be private and underscore leading, right?

Reply:
It is! It can be done w/o problems, but they use that approach on other contracts so I think we should stick to that way.
Here an exaple:
https://github.com/ethereum-optimism/optimism/blob/051db548a7f968b8c1b88da7135d0bf9cbea4954/packages/contracts-bedrock/src/L2/L2ToL2CrossDomainMessenger.sol#L48

They use `internal` for them, without an underscore

Reply:
Hmm let's do our best practices on this PR, so maybe we can slowly improve the other contracts

Reply:
Is fine if we need to revert them, the changes will still serve as an example